examples/test-umbrella/Readme.md

```diff
@@ -28,7 +28,7 @@ Start the topology on Mininet.
 $ cd endeavour && python examples/test-umbrella/mininet/simple_sdx.py
 ```
     
-In another terminal session.
+In another terminal session. Before run ryu-manager, correct the path in umbrella/fabric.cfg where the cloned endeavour directory is placed.
```

Comment:
@jhhan Jong, I think a relative path can be used so that no one has to specify the complete absolute path.
That is, instead of `ryu-manager umbrella/umbrella.py --config-file umbrella/fabric.cfg`, write `cd endeavour && ryu-manager umbrella/umbrella.py --config-file umbrella/fabric.cfg`
Am I missing something?
Can you please fix Readme.md as per above?